### PR TITLE
Fixed #679

### LINKED ISSUE #679: MDTextFieldRound color problem in Dark Theme
### Description of the Bug

when using `MDTextFieldRound` in dark theme. the icon color has changing when on focus.

### Code and Logs

```python
from kivymd.app import MDApp
from kivy.lang import Builder

KV = """
AnchorLayout:
	MDTextFieldRound:
		icon_right: "android"
		hint_text: "hint text"
		size_hint_x: .7

"""


class Example(MDApp):
	def build(self):
		self.theme_cls.theme_style = "Dark"
		return Builder.load_string(KV)


Example().run()

```

### Screenshots

color-problem

### Versions

* OS: Debian 10
* Python: v3.9.0
* Kivy: v2.0.0rc4
* KivyMD: master

### PR DESCRIPTION
**Pull request by shashi278**
Fixed #679

### CODE REVIEW

**Review by kulothunganug — CHANGES_REQUESTED** [review]
pls also do it for left icon

**Comment by shashi278** [comment]
I don't see a left icon there. Please provide a minimal example which shows the issue.

**Comment by kulothunganug** [comment]
> I don't see a left icon there. Please provide a minimal example which shows the issue.

```py
from kivymd.app import MDApp
from kivy.lang import Builder

KV = """
AnchorLayout:
	MDTextFieldRound:
		icon_right: "android"
		icon_left: "key"
		hint_text: "hint text"
		size_hint_x: .7

"""


class Example(MDApp):
	def build(self):
		self.theme_cls.theme_style = "Dark"
		return Builder.load_string(KV)


Example().run()
```

File: kivymd/uix/textfield.py [hunk 1]
```diff
@@ -1271,6 +1271,7 @@ def __init__(self, **kwargs):
         self._lbl_icon_right = MDIcon(theme_text_color="Custom")
         super().__init__(**kwargs)
         self.cursor_color = self.theme_cls.primary_color
+        self.icon_left_color = self.theme_cls.text_color
```
Comment (HeaTTheatR) [inline kivymd/uix/textfield.py:1274]: This only works if we set and remove focus from the text field. By default, the icon color is still not set correctly:

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: kivymd/uix/textfield.py (hunk 1 above)
Comment (shashi278) [reply kivymd/uix/textfield.py:1274]: I'm sure there's some patch mismatch while testing on your end. It's working fine.
ezgif com-video-to-gif(1)

Comment (HeaTTheatR) [reply kivymd/uix/textfield.py:1274]: Yes, sorry.

Comment (shashi278) [reply kivymd/uix/textfield.py:1274]: No worries ;)

**Pull request merged**